src/Selenium2Library/keywords/_element.py

```diff
@@ -680,6 +679,31 @@ def remove_location_strategy(self, strategy_name):
         """
         self._element_finder.unregister(strategy_name)
 
+    def element_attribute_should_be(self, locator, attr):
+        """Checks whether element has attribute with value
+
+        Usage Example:
+        | Element Attribute Should Be | xpath=//input | key=value |
+        """
+        self._info("Checking attributes '%s' for element '%s'" % (
+                                                    attr, locator))
+        try:
+            key, value = attr.split("=", 1)
+        except ValueError:
+            raise AssertionError(
+                    "Attribute %s does not match pattern key=value" % attr)
+
+        el = self._element_find(locator, True, True)
+        retval = el.get_attribute(key)
+
+        if retval is None:
```

Comment:
It is just extra error that element does not have attribute. I wanted prevent case like 
| Element Attribute Should Be |  css=div |  text=${None}  |
bc when element does not have attribute get_attribute returns None.
To check whether attribute is empty it should be 
| Element Attribute Should Be |  css=div |  text=${EMPTY}  |
and for checking attributes like disabled and autofocus it should be 
| Element Attribute Should Be |  css=input |  autofocus=true |